Movies-MVC/Movies-MVC/Controllers/TiserViewController.swift

```diff
@@ -0,0 +1,45 @@
+// TiserViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+import WebKit
+
+// Экран тизера
+final class TiserViewController: UIViewController {
+    // MARK: - Private Visual Componrnts
+
+    private lazy var webView: WKWebView = {
+        let webWiew = WKWebView(frame: CGRect(
+            origin: CGPoint(x: view.bounds.minX, y: 100),
+            size: view.bounds.size
+        ))
+        return webWiew
+    }()
+
+    var videoUrl = String()
```

Comment:
```suggestion
    var videoUrlString = String()
```
var выше приватного var